department-of-veterans-affairs/vets-website
# [22394] cheetah update facility phone

## Description
Some facilities have specific phone numbers set up for COVID-19 vaccine scheduling instead of routing Veterans through their main facility line.

Facility Locator ticket #23095

To Test:

1. Navigate to a vaccine appointment eg. [http://localhost:3001/health-care/schedule-view-va-appointments/appointments/va/4227b97525d391317233f4d129d53106](http://localhost:3001/health-care/schedule-view-va-appointments/appointments/va/4227b97525d391317233f4d129d53106)
2. Notice the number listed should be the covid line `307-778-7580` and not the main line `307-778-7550`

## Testing done
Local and unit

## Screenshots
<img width="697" alt="image" src="https://user-images.githubusercontent.com/8315447/117329913-1effad80-ae63-11eb-9444-f7036cacc374.png">


## Acceptance criteria
- [ ] Changes are behind the va_online_scheduling_cheetah feature flag
- [ ] Given vaccine flow, When facility has a vaccine scheduling phone number, Then display as contact number
- [ ] Given vaccine flow, When facility does not have a vaccine scheduling phone number, Then display main facility number
- [ ] Given primary or specialty flow, When facility has a vaccine scheduling phone number, Then display main facility phone number (i.e. don't show vaccine scheduling phone number when not in the vaccine flow)

## Definition of done
